SETUP.md

```diff
@@ -77,7 +77,16 @@ After setup, read about our [code styleguide](./STYLEGUIDE.md), our [test suites
 
 ### OS X Monterey - including Apple Silicon (M1)
 
-These steps are for OSX devices, including Apple Macbooks running on [Apple Silicon (M1)](https://en.wikipedia.org/wiki/Apple_silicon#M_series), which requires special consideration in order to run natively (without [Rosetta](https://en.wikipedia.org/wiki/Rosetta_(software))). These steps may need to change over time as 3rd party tools update to have versions compatible with the new architecture.
+These steps are for OSX devices, including Apple Macbooks running on [Apple Silicon (M1)](https://en.wikipedia.org/wiki/Apple_silicon#M_series). At this time, if you are using an M1 Macbook, we strongly recommend using Rosetta to set up an Intel-based development environment vs. trying to make things work with the ARM-based Apple Silicon environment.
+
+These steps may need to change over time as 3rd party tools update to have versions compatible with the new architecture.
+
+0. _(M1 Mac users only)_ Install Rosetta 2.
+
+  - Check if Rosetta is already installed: `/usr/bin/pgrep -q oahd && echo Yes || echo No`
+  - If not, install Rosetta using
+    - `softwareupdate --install-rosetta` (launches the Rosetta installer) or
+    - `/usr/sbin/softwareupdate --install-rosetta --agree-to-license` (skips installer and license agreement)
 
 1. Open your Terminal. These steps assume you are using **zsh**, the default shell for OSX.
```

Comment:
Is there a step needed to ensure you open your terminal with rosetta?

Reply:
For example, I have Rosetta installed, but my terminal is ARM.

![image](https://user-images.githubusercontent.com/7014619/199378242-c653e42d-d5eb-43d5-9624-1e06925f9059.png)

Reply:
Here's a [reference](https://www.bigbinary.com/books/learn-rubyonrails-book/setting-up-mac) to enable Rosetta. Does this work for you, @cat5inthecradle ?